packages/components/src/Spinner/Spinner.mdx

```diff
@@ -41,16 +35,33 @@ button or card contents) or when inline with content
   <Spinner size="small" />
 </Playground>
 
-## Inline
+### Inline
 
-Use `inline` to render the spinner inline with content.
+Use `inline` to render the spinner inline with content, such as when it has a leading
+piece of text content that adds context to what loading is happening.
 
 <Playground>
   <Text>
     Uploading... <Spinner size="small" inline={true} />
   </Text>
 </Playground>
 
+## Accessibility
+
+The `Spinner` should announce itself to users of assistive technology such as screenreaders.
+
+The following built-in properties help `Spinner` convey its' role effectively:
+
+- `role="alert"` allows the `Spinner` to be announced as an alert feedback element
+- `aria-busy={true}` informs the user that the view is busy, setting the expectation that they cannot interact for the time being
+- `aria-label="loading"` reads out "Loading" to assistive teach, adding context as to _why_ the view is busy
+
+## Props
+
+<Props of={Spinner} />
```

Comment:
Our other components have the props list immediately following the first example. For consistency I think it would be good to keep these there.